build.sh

```diff
@@ -356,6 +360,25 @@ copy_dma_firmware()
     cp "${PROPRIETARY_FIRMWARE_DIR}/imx/sdma/sdma-imx6q.bin" "${PROPRIETARY_FIRMWARE_OUTPUT_DIR}/imx/sdma/"
 }
 
+
+bootscript_build()
```

Comment:
I now realize that the bootscript build might have a better home in the um-u-boot repository.

Reply:
removed it